test/tutorial/cpp/ddt/graph.cpp

```diff
@@ -0,0 +1,39 @@
+#include "hclib_cpp.h"
+#include <unistd.h>
+
+/*
```

Comment:
I think if this material is intended to be used in tutorials, every file should have a descriptive comment like the one below at the top.

Not necessarily with a picture of the task graph, but at a minimum there should be a comment describing what the file implements (even for files that have a semi-descriptive filename, e.g. "fib").